SimplenoteIntents/IntentHandler.swift

```diff
@@ -61,3 +51,32 @@ extension IntentHandler: NoteWidgetIntentHandling {
         return WidgetNote(identifier: note.simperiumKey, display: note.limitedTitle)
     }
 }
+
+extension IntentHandler: ListWidgetIntentHandling {
+    func provideTagOptionsCollection(for intent: ListWidgetIntent, with completion: @escaping (INObjectCollection<WidgetTag>?, Error?) -> Void) {
+        guard let tags = try? WidgetDataController(coreDataManager: coreDataManager).tags() else {
+            completion(nil, WidgetError.appConfigurationError)
+            return
+        }
+
+        // Return collection to intents
+        let collection = tagNoteInObjectCollection(from: tags)
+        completion(collection, nil)
+    }
+
+    private func tagNoteInObjectCollection(from tags: [Tag]) -> INObjectCollection<WidgetTag> {
+        let allNotesWidgetTag: [WidgetTag] = [WidgetTag(identifier: WidgetConstants.allNotesIdentifier, display: Constants.allNotesDisplay)]
+        let fetchedWidgetTags = tags.map({ tag in
+            WidgetTag(identifier: tag.name ?? "Unamed Tag", display: tag.name ?? "Unamed Tag")
```

Comment:
Nipticky: Localized String in the placeholder. Plus: `Unnamed` or `Unamed` ? double N?